x/superfluid/README.md

```diff
@@ -223,16 +223,6 @@ osmo-basepair pool of an asset. The multiplier is set once per epoch, at
 the beginning of the epoch. In the future, we will switch this out to
 use a TWAP instead.
 
-### State changes
-
-The state of superfluid module state modifiers are classified into below
-categories.
-
-- [Proposals](07_proposals.md)
-- [Messages](03_messages.md)
-- [Epoch](04_epoch.md)
-- [Hooks](06_hooks.md)
-
```

Comment:
I do not think this has to be deleted, maybe change these links to the ones found here: https://docs.osmosis.zone/osmosis-core/modules/superfluid/#state-changes ?

Reply:
other reviewers will know more on this